go.sum

```diff
@@ -227,7 +225,7 @@ github.com/hashicorp/go-checkpoint v0.5.0/go.mod h1:7nfLNL10NsxqO4iWuW6tWW0HjZuD
 github.com/hashicorp/go-cleanhttp v0.5.0/go.mod h1:JpRdi6/HCYpAwUzNwuwqhbovhLtngrth3wmdIIUrZ80=
 github.com/hashicorp/go-cleanhttp v0.5.1 h1:dH3aiDG9Jvb5r5+bYHsikaOUIpcM0xvgMXVoDkXMzJM=
 github.com/hashicorp/go-cleanhttp v0.5.1/go.mod h1:JpRdi6/HCYpAwUzNwuwqhbovhLtngrth3wmdIIUrZ80=
-github.com/hashicorp/go-getter v1.5.1 h1:LZ49OxqBBtdKJymlpX7oTyqGBQRg4xxQDyPW4hzoZqM=
+github.com/hashicorp/go-getter v1.5.1 h1:lM9sM02nvEApQGFgkXxWbhfqtyN+AyhQmi+MaMdBDOI=
```

Comment:
Did you update the `go.sum` manually ? 
The `go mod download` command fails with
```
$ go mod download 
verifying github.com/hashicorp/go-getter@v1.5.1: checksum mismatch
	downloaded: h1:lM9sM02nvEApQGFgkXxWbhfqtyN+AyhQmi+MaMdBDOI=
	go.sum:     h1:LZ49OxqBBtdKJymlpX7oTyqGBQRg4xxQDyPW4hzoZqM=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

Otherwise the build seems fine

Reply:
Did a short refactoring as some of the terraform modules moved. 
I also used go get -u to update this time.

Reply:
Will test asap, thanks for the update

Reply:
Thank you both for your work. Is this good to merge now?

Reply:
Though a README update is needed too

Reply:
Hello @Tyron2k, sorry for the late answer but we tested it and it seems ok for us.
Can you just squash your commit in order to let a clean git history?

Thank you